guides/common/modules/proc_configuring-repositories-proxy.adoc

```diff
@@ -30,10 +30,13 @@ Use this procedure to enable the repositories that are required to install {Prod
 ----
 # dnf module enable {dnf-module}
 ----
+
 +
 [NOTE]
 ====
-include::snip_dnf-module-enable-note.adoc[]
+If there is any warning about conflicts with Ruby or PostgreSQL while enabling `{dnf-module}` module, see {InstallingServerDocURL}Troubleshooting_Ruby_{project-context}[Troubleshooting Ruby] or
```

Comment:
I'm not too famliar with this, but since it's in the same document, should this use `xref`?

Reply:
Addressed :+1: